### PR TITLE
Js linter

### PR DESCRIPTION
**Pull request by mullinmax**
## Description
 - Added java script/css/html linter prettier
 - ran `pre-commit run --all` and corrected all issues

### CODE REVIEW

File: src/ScoreTrack.py [hunk 1]
```diff
@@ -188,17 +190,17 @@ def place_machine_scores():
                 shadowRam[initial_start + 1] = 64
                 shadowRam[initial_start + 2] = 64
 
-    elif S.gdata["HighScores"]["Type"] is 9:
-        #system 9, copy in scores, no intiials 
+    elif S.gdata["HighScores"]["Type"] == 9:
+        # system 9, copy in scores, no intiials
```
Comment (mullinmax) [inline src/ScoreTrack.py:194]: FYI: `==` checks if the values are equal, `is` checks the object in memory is the same. Basically you want to use `is` when comparing to None and otherwise use `==`

**Comment by mullinmax** [comment]
Tons of changes in here, but 99% of them are just formatting. I don't think I made any changes that would change functionality but if anything would change something it would be in this commit you can see here:

https://github.com/warped-pinball/vector/pull/52/commits/2bf251f426e8be9e3250d9a292f70c29896ce362

I made a note about `is`/`==`. I think I got all the changes right, but I don't have a way to test myself. It sorta depends on if those were edge cases or not. I think if the above link looks ok (it's the `pre-commit fixes` commit in this PR) then I think the rest should be fine without heavy scrutiny.

**Review by paulmullin — DISMISSED** [review]

**Review by paulmullin — APPROVED** [review]

File: src/ScoreTrack.py (hunk 1 above)
Comment (paulmullin) [reply src/ScoreTrack.py:194]: roger that!
